data/ecosystem/vendors.yaml

```diff
@@ -216,6 +216,13 @@
   contact: ''
   oss: true
   commercial: true
+- name: OpenObserve
+  distribution: false
+  nativeOTLP: true
+  url: 'https://openobserve.ai'
+  contact: ''
```

Comment:
Please provide a contact, we use this to get in touch in case of changes or issues. A GitHub handle to tag is ideal, email is also possible